doc/source/index.rst

```diff
@@ -0,0 +1,20 @@
+.. Tenks documentation master file, created by
+   sphinx-quickstart on Thu Sep 20 15:43:21 2018.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to Tenks's documentation!
+=================================
+
+Tenks is a utility that manages virtual bare metal clusters for development and
+testing purposes.
+
+.. toctree::
+   :maxdepth: 2
+
+   architecture
+   configuration
+   development
+   install
+   limitations
+   run
```

Comment:
Sounds more reasonable - perhaps architecture should come between run and development though, since it's not required to be understood in order to deploy a cluster? Then the first three sections would form a sort of 'quick-start' guide.